modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -117,26 +117,24 @@ For more information about using variables in {ay} scripts, see xref:client-conf
 These are the most important sections in the {ay} installation file for installing with {productname}:
 
 * ``<add-on>`` allows to add child channels to the installation
+** See https://doc.opensuse.org/projects/autoyast/#Software-Selections-additional with an ``<add-on>`` example
```

Comment:
I do not like this new layout technique ;) I think we can simply continue without a formal subitem:

```suggestion
* ``<add-on>`` allows to add child channels to the installation.
  For an example, see https://doc.opensuse.org/projects/autoyast/#Software-Selections-additional.
```